plugins/processors/tag_limit/tag_limit.go

```diff
@@ -0,0 +1,87 @@
+package taglimit
+
+import (
+	"fmt"
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/plugins/processors"
+	"log"
+)
+
+var (
+	keepTags map[string]string
```

Comment:
nitpick: put this var in the `TagLimit` struct

Reply:
done